std_msgs/Header header
string soma_roi_id
string methods
int64[] code_book
string[] graphlet_book
